packer/docs-rs-builder/builder.pkr.hcl

```diff
@@ -9,6 +9,7 @@ packer {
 
 data "amazon-parameterstore" "revision" {
   name = "/docs-rs/builder/sha"
+  region = "us-east-1"
 }
```

Comment:
The formatting is broken here. We run `terraform fmt` in CI, do we want to do the same for Packer?